test/WebJobs.Script.Tests/RuntimeInformation/TestSystemRuntimeInformation.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests
+{
+    public class TestSystemRuntimeInformation : ISystemRuntimeInformation
+    {
+        public Architecture GetOSArchitecture()
+        {
```

Comment:
Use TestEnvironment instead

Reply:
Resolving since we'll discuss this offline.

Reply:
lets keep this active until we sync up as a reminder to follow up